examples/example1.py

```diff
@@ -28,10 +29,10 @@
 
 import pyhgvs as hgvs
 import pyhgvs.utils as hgvs_utils
-from pygr.seqdb import SequenceFileDB
+from pyfaidx import Fasta
 
-# Read genome sequence using pygr.
-genome = SequenceFileDB('hg19.fa')
+# Read genome sequence using pyfaidx.
+genome = Genome('/tmp/hg19.fa')
```

Comment:
Does `Genome()` allow relative paths? If so, I think the relative paths may be more convenient for people to try out the library. If it requires absolute paths, then this is a very reasonable change to make.